sql/item_usable.sql

```diff
@@ -1623,14 +1623,14 @@ INSERT INTO `item_usable` VALUES (5820,'darkling_bolt_quiver',1,4,55,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5821,'fusion_bolt_quiver',1,4,55,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5822,'dweomer_bullet_pouch',1,4,55,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5823,'oberon_bullet_pouch',1,4,55,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5824,'lucid_potion_i',1,4,30,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5825,'lucid_potion_ii',1,4,31,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5826,'lucid_potion_iii',1,4,31,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5827,'lucid_ether_i',1,4,0,32,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5828,'lucid_ether_ii',1,4,0,33,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5829,'lucid_ether_iii',1,4,0,33,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5830,'lucid_elixir_i',1,4,34,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (5831,'lucid_elixir_ii',1,4,34,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (5824,'lucid_potion_i',1,1,30,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (5825,'lucid_potion_ii',1,1,31,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (5826,'lucid_potion_iii',1,1,31,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (5827,'lucid_ether_i',1,1,0,32,0,0,0,0);
```

Comment:
the three inserts for ethers are missing animation (probably the value in animation time is supposed to be in the animation column)

Reply:
omg ty i didnt notice <3